binding/python/raftify.pyi

```diff
@@ -71,6 +72,34 @@ class ReadOnlyOption:
     in that case.
     """
 
+class Level:
+    DEBUG: Final
```

Comment:
When I ran mypy on my local computer, I encountered the following error: It cannot infer the types of the variants. 

In addition to the Final qualifier, you need to specify the type.

```
❯ mypy raftify.pyi 
raftify.pyi:76: error: Type in Final[...] can only be omitted if there is an initializer  [misc]
```